assets/js/state/hosts.js

```diff
@@ -59,6 +59,22 @@ export const hostsListSlice = createSlice({
         return host;
       });
     },
+    setHostDeregistering: (state, action) => {
+      state.hosts = state.hosts.map((host) => {
+        if (host.id === action.payload.id) {
+          host.deregistering = true;
```

Comment:
@CDimonaco left a [good comment on my PR](https://github.com/trento-project/web/pull/1587#discussion_r1257914093) about making this reducer function more pure that I think applies here also